GOVERNANCE.md

```diff
@@ -1,12 +1,34 @@
 # Governance
 
-## License
+## GUAC community
 
-The project will use [Apache 2.0 license](LICENSE) for code.
+The GUAC community (including GUAC OpenSSF meetings, website  and [github organization](https://github.com/guacsec/) administration) is home to several projects focused around solving problems related to the consumption and utilization of supply chain metadata at scale. The community contains "Core projects" which emcapsulate large bodies of work where a separate governance of code and process makes sense, while retaining requirements around aspects such as licensing and code of conduct.
+
+Thus the governance is structured into the "GUAC community maintainers" team and "GUAC core projects" maintainers teams.
+
+### GUAC community index
+
+- GUAC Community:
+  - governance
+  - guac-landing
+  - guac-site
+  - .allstar
+  - .github
+- GUAC Core Project: Graph for Understanding Artifact Composition (GUAC) ([MAINTAINERS.guac](MAINTAINERS.guac))
+  - guac-docs
```

Comment:
ben: should be at the community level unless we want to have a different docs site